ocaml/xapi/xapi_vm_migrate.ml

```diff
@@ -1577,6 +1577,7 @@ let migration_type ~__context ~remote =
 let assert_can_migrate ~__context ~vm ~dest ~live ~vdi_map ~vif_map ~options
     ~vgpu_map =
   assert_licensed_storage_motion ~__context ;
+  Xapi_vm_helpers.assert_dmc_compatible ~__context ~vm ;
```

Comment:
You also need this in `VM.pool_migrate` in message_forwarding, as this function isn't called for pool_migrate.